### PR TITLE
feat: auto-suggest address

### LINKED ISSUE #69: Auto-suggest of addresses
I've recently implemented some function for Chat which spots when you are typing a postal address and suggests it:

image

This works by:

- checking if we know the user's postcode
- querying the server for the list of postal addresses in that postcode
- spotting when you start typing one of them
- showing a suggestion

I've experimented with various ways of showing this to the user:

1. Via `Popper` (the current approach).  This gets very low interaction.  I would guess that is because it is outside the textbox and therefore some people won't even see it.
2. Via a popup modal.  This gets a lot more interaction (as you would expect), but more cancels than people accepting it.  Some of this may be that people react to interrupting popups by closing them reflexively.  Popups are annoying.

I'd like to try an approach which looks like standard autocomplete, so that the suggestion was just the address (without any text) and was shown right next to the cursor.  Then people would see it.  If they don't select it, but typing the exact same match then we should treat that as a selection.

Can you find a way to implement that?

### PR DESCRIPTION
**Pull request by untael**
Closes #69 

Implemented in the same way as mentions in `OurAtTa.vue` component (based on floating-vue library). Clicking on suggested address will autofill the address in the textarea.

### CODE REVIEW

**Comment by edwh** [comment]
The deploy failed.  If I run `npm run dev` then I see an error that suggests that this doesn't work with SSR:

```
500
Element is not defined

at C:\Users\edwar\PhpstormProjects\iznik-nuxt3\node_modules\floating-vue\dist\floating-vue.mjs:1094:30
at async ViteNodeRunner.runModule (/C:/Users/edwar/PhpstormProjects/iznik-nuxt3/node_modules/vite-node/dist/client.mjs:362:5)
at async ViteNodeRunner.directRequest (/C:/Users/edwar/PhpstormProjects/iznik-nuxt3/node_modules/vite-node/dist/client.mjs:346:5)
at async ViteNodeRunner.cachedRequest (/C:/Users/edwar/PhpstormProjects/iznik-nuxt3/node_modules/vite-node/dist/client.mjs:189:14)
at async ViteNodeRunner.dependencyRequest (/C:/Users/edwar/PhpstormProjects/iznik-nuxt3/node_modules/vite-node/dist/client.mjs:233:12)
at async C:\Users\edwar\PhpstormProjects\iznik-nuxt3\virtual:nuxt:C:\Users\edwar\PhpstormProjects\iznik-nuxt3\.nuxt\floating-vue.mjs:2:31
at async ViteNodeRunner.runModule (/C:/Users/edwar/PhpstormProjects/iznik-nuxt3/node_modules/vite-node/dist/client.mjs:362:5)
at async ViteNodeRunner.directRequest (/C:/Users/edwar/PhpstormProjects/iznik-nuxt3/node_modules/vite-node/dist/client.mjs:346:5)
at async ViteNodeRunner.cachedRequest (/C:/Users/edwar/PhpstormProjects/iznik-nuxt3/node_modules/vite-node/dist/client.mjs:189:14)
at async ViteNodeRunner.dependencyRequest (/C:/Users/edwar/PhpstormProjects/iznik-nuxt3/node_modules/vite-node/dist/client.mjs:233:12)
```

Can you have a look at this, please?

**Comment by untael** [comment]
Issue was caused by unstable outdated version of `floating-vue` library. I bumped it. Should work fine for now.

File: nuxt.config.ts [hunk 1]
```diff
@@ -158,17 +158,7 @@ export default defineNuxtConfig({
     extractCSS: true,
   },
 
-  modules: ['@pinia/nuxt'],
-
-  buildModules: [
```
Comment (untael) [inline nuxt.config.ts:163]: Removed because it was deprecated.
Reference: https://nuxt.com/docs/guide/concepts/modules#add-nuxt-modules

**Comment by edwh** [comment]
> Should work fine for now.

It does - thanks,

**Pull request merged**